volatility3/framework/symbols/windows/extensions/mft.py

```diff
@@ -0,0 +1,21 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+from volatility3.framework import objects
+
+
+class MFTEntry(objects.StructType):
+    """This represents the base MFT Record"""
+
+    def get_signature(self) -> str:
+        signature = self.Signature.cast('string', max_length = 4, encoding = 'latin-1')
```

Comment:
I'm ok with this since the `get_full_name` method below is still needed, but generally I'd try to discourage class overrides just for a single method that's only a line long.  So with something like this, where the signature which doesn't really need to be a string, but could be matched against an `int` for example (`== 0xDEADBEEF`), I'd probably do it in the code directly.  As I say, fine here because we need the file anyway...  5:)